lib/streamlit/runtime/credentials.py

```diff
@@ -67,42 +66,49 @@ def email_prompt() -> str:
 
 
 def _send_email(email: str) -> None:
-    """Send the user's email to segment.io, if submitted"""
+    """Send the user's email for metrics, if submitted"""
     import requests
 
     if email is None or "@" not in email:
         return
 
+    metrics_url = ""
+    try:
+        response_json = requests.get(
+            "https://data.streamlit.io/metrics.json", timeout=2
+        ).json()
+        metrics_url = response_json.get("url", "")
+    except Exception:
+        _LOGGER.error("Failed to fetch metrics URL")
+        return
+
     headers = {
-        "authority": "api.segment.io",
         "accept": "*/*",
         "accept-language": "en-US,en;q=0.9",
-        "content-type": "text/plain",
+        "content-type": "application/json",
         "origin": "localhost:8501",
         "referer": "localhost:8501/",
     }
 
-    dt = f"{datetime.now(timezone.utc).isoformat()}+00:00"
-
     data = {
         "anonymous_id": None,
-        "context": {
-            "library": {"name": "analytics-python", "version": "2.2.2"},
-        },
         "messageId": str(uuid4()),
-        "timestamp": dt,
```

Comment:
Fivetran sends the `_created` timestamp automatically, but lmk if you still want this field